sdk/pom.xml

```diff
@@ -186,7 +186,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.13.3</version>
+      <version>2.14.1</version>
```

Comment:
Try to run application.
We can't update jackson dependency as is to `2.14.1`.
Assume that it requires to have some updates starting from Sparkz.
For now we may keep it as is.

Reply:
Try to run application.